core/src/genesis.rs

```diff
@@ -17,18 +17,16 @@
 use time;
 
 use core;
-use consensus::MINIMUM_DIFFICULTY;
+use consensus;
 use core::target::Difficulty;
 use global;
 
-/// Genesis block definition. It has no rewards, no inputs, no outputs, no
-/// fees and a height of zero.
-pub fn genesis() -> core::Block {
-	let proof_size = global::proofsize();
-	let empty_hash = core::hash::Hash([0x0; 32]);
+/// Genesis block definition for development networks. The proof of work size
+/// is small enough to mine it on the fly, so it does not contain its own
+/// proof of work solution. Can also be easily mutated for different tests.
+pub fn genesis_dev() -> core::Block {
```

Comment:
Is this `UserTesting`? I think we should define the nonce and pow for this one (otherwise it can take a couple of minutes to startup a UserTesting node (mining or non-mining).

Reply:
There is a nonce for this in the globals. With a proper nonce, it's just a single try so it doesn't seem we would need the solution?

Reply:
Oh I see - thought we still needed the pow solution in addition to the nonce in there. :+1: